ITR/data/base_providers.py

```diff
@@ -247,7 +248,10 @@ def _convert_projections_to_series(self, company: ICompanyData, feature: str,
         :return: pd.Series
         """
         company_dict = company.dict()
-        production_units = company_dict[self.column_config.PRODUCTION_METRIC]['units']
+        try:
+            production_units = company_dict[self.column_config.PRODUCTION_METRIC]['units']
+        except TypeError:
+            breakpoint()
```

Comment:
we don't propagate `TypeError` further, is it intended?

Reply:
This is debugging code that should be removed/vestiges of the big Pydantic cleanup that needs some small cleanups.  Good catch.